README.md

```diff
@@ -115,10 +115,10 @@ Can be a date parseable string, a millisecond timestamp, or a Date object. The c
  <calendar-heatmap endDate="2018-9-22" ... />
 ```
 
-### **colorRange** - `color-range` 
-A Array of 5 strings which represents the colors of the progression. The color at `colorRange[0]` will always represent the values for a `count: 0`. The others are automatically distributed over the maximum value of count, unless you specify `max` props. Default value is equal to the example.
+### **rangeColor** - `range-color` 
+A Array of 5 strings which represents the colors of the progression. The color at `rangeColor[0]` will always represent the values for a `count: 0`. The others are automatically distributed over the maximum value of count, unless you specify `max` props. Default value is equal to the example.
 ``` html
- <calendar-heatmap :range-color="['ebedf0', '#c0ddf9', '#73b3f3', '#3886e1', '#17459e']" ... />
+ <calendar-heatmap :rangeColor="['ebedf0', '#c0ddf9', '#73b3f3', '#3886e1', '#17459e']" ... />
```

Comment:
Thanks for contribution. I'am not agree with your correction, to my knowledge we can not write the html attribute with capital letters.

And see this [rules](https://vuejs.org/v2/style-guide/#Prop-name-casing-strongly-recommended) from vue official style guide

Reply:
Thanks for pointing this out. You are of course correct. At the time I thought `tooltip-unit` was not working and could only get `tooltipUnit` to work, but must have been on the wrong path!

I've update the branch to only include the minor typo changes.